conf/nginx.conf

```diff
@@ -3,6 +3,11 @@ location __PATH__/ {
 
   # Path to source
   alias __FINALPATH__/;
+  
+  # static files
+  proxy_pass http://localhost:5000;
```

Comment:
That's confusing, either use `alias` to serve static file, or use `proxy_pass` to reverse-proxy to a "dynamic app", but both at the same time make no sense

Reply:
I don't understand the difference… but I guess this is a static app ?

Reply:
I tried to remove the alias